lesson_07/objects/objects_app/src/main/java/com/codedifferently/lesson7/Jordan_Eldridge/Animal.java

```diff
@@ -1,4 +1,7 @@
-package com.codedifferently.lesson7.Jordan_Eldridge;
+package com.codedifferently.lesson7.jordan_eldridge;
```

Comment:
Folder names should also be lowercase to match the package name, and you don't need the underscore :D

Reply:
ok sweet learning alot by doing this thank you !